docs/book/src/cronjob-tutorial/testdata/project/internal/controller/suite_test.go

```diff
@@ -74,11 +74,26 @@ var _ = BeforeSuite(func() {
 
 	ctx, cancel = context.WithCancel(context.TODO())
 
+	var err error
+	/*
+		First, the autogenerated test code will add the CronJob Kind schema to the default client-go k8s scheme.
+		This ensures that the CronJob API/Kind will be used in our test controller.
+	*/
+	err = batchv1.AddToScheme(scheme.Scheme)
+	Expect(err).NotTo(HaveOccurred())
+	/*
+		After the schemas, you will see the following marker.
+		This marker is what allows new schemas to be added here automatically when a new API is added to the project.
+	*/
+
+	// +kubebuilder:scaffold:scheme
+
 	/*
-		First, the envtest cluster is configured to read CRDs from the CRD directory Kubebuilder scaffolds for you.
+		The envtest cluster is configured to read CRDs from the CRD directory Kubebuilder scaffolds for you.
```

Comment:
```suggestion
		/*
    The envtest environment is configured to load Custom Resource Definitions (CRDs) from the specified directory.
    This directory is scaffolded by Kubebuilder and contains the CRD manifests for the APIs defined in the project.
    The `CRDInstallOptions` ensures the specified scheme is used when installing these CRDs into the test environment.
    This setup enables the test environment to recognize and interact with the custom resources defined by these CRDs.
*/
```

The original text does not seems very accurate , I think we would need to improve it as above

Reply:
will add to docs PR